### DISCUSSION TITLE
Steam deck controls are unfinished and not fully bound

### DISCUSSION THREAD

**Discussion by Rehexr** [FakeFishGames/Barotrauma#9453] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Simply put, the steam deck controls dont currently fully allow for all actions, or simply make no sense.
This especially comes to note in the fact that space is bound to the ybax buttons and the sprint is stuck as a pushdown joystick, which is impossible to use practically or comfortably.
On another thing, g is currently bound to exactly no buttons, making it impossible to move bodies without changing settings.
Personally id mostly reccomend switching out the space for the joystick pushdown, for their only occasional use and upping the aim assist significantly, for it to be useable.
As a last thing it seems that the settings for the hotbar are either incredibly confusing or just unusable properly. 
Pretty much all of this requires some degree of changing because otherwise the experience on a steam deck just control wise doesnt work.

### Reproduction steps

Try the control scheme on a steam deck system

### Bug prevalence

Happens every time I play

### Version

0.18.12.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
We don't support Steam Deck at the moment, so this is more of a feature request than a bug report. Moving to discussions.

**Comment by liammcdermott** [comment]
@Rehexr I already have a discussion in for this, please upvote it. I really hope support for this game makes it onto Steam Deck! Personally, I'm holding off purchasing the game until it's Verified.

My suggestion: https://github.com/Regalis11/Barotrauma/discussions/9313

**Reply by Rehexr** [reply]
Yea wasnt exactly by choice that this isnt an issue but considered a suggestion now, just kinda assumed that its counted compaitble that itd be counted as supported. Either way voted your discussion up

**Reply by liammcdermott** [reply]
I noticed this was converted from an issue, no worries. Sorry if my reply sounded grumpy, that was not my intention!
